build-aux/create-homebrew-formula.pl

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use LWP::Simple;
+use Crypt::Digest::SHA256 qw(sha256_hex);
+use Data::Dumper;
+
+my @build_depends = qw(luarocks pkg-config);
+my @homebrew_depends = qw(expat fontconfig harfbuzz icu4c libpng lua openssl@1.1 zlib);
+
+my %not_rocks = map {$_ => 1} qw(lua bit32 compat53);
+my %rocks = load_rockspec();
+
+$rocks{"luaexpat"} = "1.3.0"; # 1.3.3 is on luarocks, but there's no source release!
```

Comment:
Sure there is:

https://github.com/tomasguisasola/luaexpat/archive/v1.3.3.tar.gz

These source locations are listed in the rockspecs.